libs/libcommon/src/libcommon/constants.py

```diff
@@ -7,18 +7,19 @@
 QUEUE_TTL_SECONDS = 604_800  # 7 days
 
 PROCESSING_STEP_CONFIG_NAMES_VERSION = 1
-PROCESSING_STEP_CONFIG_PARQUET_VERSION = 3
-PROCESSING_STEP_CONFIG_SIZE_VERSION = 1
-PROCESSING_STEP_CONFIG_INFO_VERSION = 1
-PROCESSING_STEP_DATASET_INFO_VERSION = 1
-PROCESSING_STEP_DATASET_PARQUET_VERSION = 1
-PROCESSING_STEP_DATASET_SIZE_VERSION = 1
-PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 1
+PROCESSING_STEP_CONFIG_PARQUET_VERSION = 4
+PROCESSING_STEP_CONFIG_SIZE_VERSION = 2
+PROCESSING_STEP_CONFIG_INFO_VERSION = 2
+PROCESSING_STEP_DATASET_INFO_VERSION = 2
+PROCESSING_STEP_DATASET_PARQUET_VERSION = 2
+PROCESSING_STEP_DATASET_SIZE_VERSION = 2
+PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 2
 PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_STREAMING_VERSION = 1
 PROCESSING_STEP_PARQUET_AND_DATASET_INFO_VERSION = 1
-PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_PARQUET_VERSION = 1
+PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_PARQUET_VERSION = 3
+PROCESSING_STEP_CONFIG_PARQUET_AND_INFO_VERSION = 1
 PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_STREAMING_VERSION = 2
```

Comment:
I'm not sure if I should increment `PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_STREAMING_VERSION`. It's now (https://github.com/huggingface/datasets-server/pull/1021) dependent on `/split-names-from-dataset-info`  which in turn is dependent on `config-info` which in turn is dependent on `config-parquet-and-info` :) But it's also dependent on `/split-names-from-streaming` which is not connected to `config-parquet-and-info`.

Reply:
so I can restore all the previous versions?

Reply:
YES 😬

Reply:
[cef5ad2](https://github.com/huggingface/datasets-server/pull/985/commits/cef5ad2652f2daa1a263e9b81a69053b5f81aad8)